### ISSUE TITLE
Fontawesome fonts not working correctly

### ISSUE THREAD

**Issue by chartjes** [opencfp/opencfp#697]
# Steps required to reproduce the problem

1. Load any page inside the site

# Expected Result

* Anything styled with Font Awesome fonts should work

# Actual Result

* They don't

I went and generated a CDN embed code linked to my email address, but we need a more permanent solution.

**Comment by mdwheele** [comment]
I'll take a look once the kiddo is asleep. I'll do a fresh install because it currently works for me, so I've probably got something else going on that's allowing that.

**Comment by mdwheele** [comment]
I think I must be missing some information. It looks like iconography from FA is working correctly:

![image](https://user-images.githubusercontent.com/2453394/33195566-bb4b1a74-d0a6-11e7-810d-1d35a0c04663.png)

![image](https://user-images.githubusercontent.com/2453394/33195570-c7365e7a-d0a6-11e7-8a3b-1e1a092c05b2.png)

Any more detail on this?

**Comment by mdwheele** [comment]
One thing I noticed when building all of this is that I had to do a *hard refresh* on my browser to pick up font-awesome stuff. One day, we could implement asset versioning and all that jazz to work around the issue, but I would rather wait until this lands in a non-EOL framework (which might support this feature by default).

Gonna close this out, but if there are more details I'm missing, I'm down to help resolve.
